src/library_webgpu.js

```diff
@@ -635,6 +635,7 @@ var LibraryWebGPU = {
     VertexStepMode: [
       'vertex',
       'instance',
+      'vertex-buffer-not-used',
```

Comment:
It should be possible in dawn.json to make this `undefined` instead of `'vertex-buffer-not-used'`, using `jsrepr`. Also nit for webgpu.h, it would probably make more sense if it were item 0.